lldb/tools/debugserver/source/debugserver.cpp

```diff
@@ -156,6 +156,7 @@ RNBRunLoopMode RNBRunLoopGetStartModeFromRemote(RNBRemote *remote) {
   return eRNBRunLoopModeExit;
 }
 
+#if defined(WITH_FBS) || defined(WITH_BKS) || defined(WITH_SPRINGBOARD)
```

Comment:
This probably got here by accident?

Reply:
I added these because `is_dot_app` is used, but only conditionally. This hides the function (and prevents the unused warning) unless one of the defines are enabled.

Reply:
Ah.. and it's used in a block with the exact same conditions?
Perhaps we should convert is_dot_app into a lambda and sink it next to its use then (inside the other #if block)?